wrappers/binaries_test.go

```diff
@@ -81,3 +81,19 @@ func (s *binariesTestSuite) TestAddSnapBinariesAndRemove(c *C) {
 
 	c.Check(osutil.FileExists(link), Equals, false)
 }
+
+func (s *binariesTestSuite) TestAddSnapBinariesCleansUpOnFailure(c *C) {
```

Comment:
Yes, you're reading this correctly.

Before this PR, this test would fail (some of the time).